control/tests/test_api.py

```diff
@@ -145,3 +145,38 @@ def test_creates_control_and_adds_to_current_user():
     assert saved_control.title == payload['title']
     assert saved_control.reference_code == payload['reference_code']
     assert user.profile.controls.all().get(id=response_control['id']) == saved_control
+
+
+### Update
+
+def call_control_update_api(user, payload, control):
+    utils.login(client, user=user)
+    url = reverse('api:control-detail', args=[control.id])
+    response = client.put(url, payload, format='json')
+    return response
+
+
+def make_update_payload():
+    return {
+        "title": "updated control",
+        "depositing_organization": "updated organization",
+    }
+
+
+def test_can_access_control_update_api_if_inspector_user():
```

Comment:
il manque un test : inspector can't make update if the control doesn't belong to them

Reply:
ok